File: VB/WindowsApplication1/MyColorEdit/MyColorListBox.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors.Popup

Namespace CustomEditors
	Public Class MyColorListBox
		Inherits ColorListBox
		Protected Overrides Function GetColorName(ByVal color As System.Drawing.Color) As String
			Return ColorLocalizer.GetLocalizedColorName(color)
		End Function
	End Class
End Namespace

File: CS/WindowsApplication1/MyColorEdit/MyColorListBox.cs
using System;
using DevExpress.XtraEditors.Popup;

namespace CustomEditors
{
    public class MyColorListBox : ColorListBox
    {
        protected override string GetColorName(System.Drawing.Color color)
        {
            return ColorLocalizer.GetLocalizedColorName(color);
        }
    }
}
